### ISSUE TITLE
How can I use .bash_profile aliases in black-screen?

### ISSUE THREAD

**Issue by Krystus** [shockone/black-screen#209]
Why I can't use .bash_profile aliases in black-screen? I have to do something or it is impossible for now?

**Comment by shockone** [comment]
They should be imported automatically. If they're not, it's a bag, and I'd need more information about your setup.

**Comment by Krystus** [comment]
I have a test release 0.36.4 on iMac 27" El Capitan 10.11.3 (15D21).

![zrzut ekranu 2016-02-02 o 10 11 40](https://cloud.githubusercontent.com/assets/15039636/12744599/66f8c6f2-c995-11e5-86bf-5abba100ca00.jpg)
![zrzut ekranu 2016-02-02 o 10 11 01](https://cloud.githubusercontent.com/assets/15039636/12744600/67464468-c995-11e5-9306-bda2a9afbf0c.jpg)

**Comment by shockone** [comment]
In Black Screen, press cmd+option+i to open developer tools, switch to the Console tab and type `process.env.SHELL` is it `/bin/bash`? 

If yes, please try to copy the file `.bash_profile` to `.bashrc` and reload Black Screen.

Please let me know if any of this helped.

**Comment by Krystus** [comment]
Yes, that was /bin/bash. After .bash_profile to .bashrc copy of aliases easy commands like desk, twc working. But take a look at my aliases in second screen... homestead up and twc.start/stop don't work at all.

![zrzut ekranu 2016-02-02 o 10 33 45](https://cloud.githubusercontent.com/assets/15039636/12745211/764f5672-c998-11e5-9212-6c5895a3e4cd.jpg)

**Comment by shockone** [comment]
There are three bugs. Good catch!

* Tokens are split at a dot. That's why `php-fpm.start` doest't work. Should be fixed soon.
* `.bash_profile` isn't loaded. It's intended for non-interactive sessions, but clearly people use it as interactive session configs too. Should be fixed soon.
* Shell's built-in functions don't work. In your case `function`, but also things like `nocorrect` and so on. Might be harder to fix.

**Comment by shockone** [comment]
Now you don't need `.bashrc`.

**Comment by ilanbiala** [comment]
Have these three issues been fixed?

**Comment by shockone** [comment]
Only the second one at the moment, @ilanbiala.

**Comment by eliorc** [comment]
I'm having the same problem with Ubuntu 18.04, Upterm 0.4.3.

All my aliases are in `~/.bashrc` and are not being loaded - meaning they don't work and obviously don't autocomplete.

Typing `process.env.SHELL` in the developer tools console shows `/bin/bash`

**Comment by zacanger** [comment]
Is this related to builtin problems in #1215?
